test-suite/Add_test.go

```diff
@@ -0,0 +1,65 @@
+package testSuite
+
+import (
+	"testing"
+
+	testUtils "github.com/replit/upm/test-suite/utils"
+)
+
+func TestAdd(t *testing.T) {
+	for _, bt := range languageBackends {
+		bt.Start(t)
+
+		var pkgs []string
+		switch bt.Backend.Name {
+		default:
+			t.Run(bt.Backend.Name, func(t *testing.T) {
+				t.Skip("no test")
+			})
+			continue
+		}
```

Comment:
this pattern will be fleshed out in individual PRs adding per-backend tests, but results in lint failures in this PR :(